src/main/java/games/strategy/engine/data/export/GameDataExporter.java

```diff
@@ -143,7 +143,6 @@ private static String technologies(final GameData data) {
     return returnValue.toString();
   }
 
-  @SuppressWarnings("unchecked")
```

Comment:
Note that the removal of this annotation causes Eclipse to report several unchecked cast warnings in this methods.  These warnings are reported in Neon.3.  @RoiEXLab, can you confirm if the warnings are still reported in Oxygen?